packages/aggregator/src/connections/EntityConnection.js

```diff
@@ -13,6 +13,22 @@ export class EntityConnection extends ApiConnection {
   constructor(session) {
     const { getAuthHeader } = session;
     super({ getAuthHeader });
+
+    session.aggregatorEntityConnectionCache = session.aggregatorEntityConnectionCache || {};
+    this.cache = session.aggregatorEntityConnectionCache;
+  }
+
+  // functionArguments should receive the 'arguments' object
+  getCacheKey = (url, query) => `${url}:${JSON.stringify(query)}`;
```

Comment:
I think it can be a bit unreliable using `JSON.stringify()` for building cache strings from objects. I'm it always outputs the keys in the same order.  We use it in our DatabaseModel class though and I think it works alright... Perhaps a 3rd party library like `object-hash` might be better? https://www.npmjs.com/package/object-hash

Reply:
True! I guess it "works alright" because probably most of the time it comes out the same, and if it doesn't, there's no massive issue other than a bit of extra lag. Still seems worth using `object-hash` though :+1: